examples/model_selection/plot_roc.py

```diff
@@ -101,7 +124,11 @@
 
 
 ##############################################################################
-# Plot ROC curves for the multiclass problem
+# Plot ROC curves for the multiclass problem using One vs. Rest classification.
+
+# Compute micro-average ROC curve and ROC area
+fpr["micro"], tpr["micro"], _ = roc_curve(y_test.ravel(), y_score.ravel())
```

Comment:
If we believe this is a valid use of the measure, should we not be offering it in the library?